packages/docs/blog/2023-03-28-estimating-rds-size.md

```diff
@@ -0,0 +1,53 @@
+---
+slug: estimating-rds
+title: Estimating RDS Size
+authors:
+  name: Reshma Khilnani
+  title: Medplum Core Team
+  url: https://github.com/reshmakh
+  image_url: https://github.com/reshmakh.png
+tags: [fhir-datastore, self-host]
+---
+
+# Estimating RDS Size
+
+Data privacy, locality, governance and [compliance](/docs/compliance) are huge issues in healthcare, and that's why we at Medplum support [self-hosting](/docs/self-host). For those running on AWS, we use Aurora RDS, which supports auto-scaling. A common question we get is - **how big is my database going to be?**
+
+:::tip
+Medplum offers a hosted offering as well as self-hosted. Instructions to [register](/docs/tutorials/register) can be found in our docs.
+:::
+
+Say for example, you have 1M active patients annually - how does that translate to database size?
+
+Unfortunately, there is no straightforward answer to that question, but the right way to think about it is to make a **FHIR resource count estimate, per patient, annually**. Applications with a lot of messaging and observation data are generally more resource intensive, while those that have visit notes and prescriptions are generally less resource intensive.
+
+Here's an example of a resource projection.
+
+| Resource Type     | Count per Patient | Total Count (M) | Avg. Size / Resource (kb) | Avg. History Length | Total Storage (GB) |
+| ----------------- | ----------------- | --------------- | ------------------------- | ------------------- | ------------------ |
+| Patients          | **\_\_**          | 1               | 5                         | 10.0                | 52.5               |
+| Encounters        | 10                | 10              | 2                         | 2.0                 | 42.0               |
+| Coverage          | 1                 | 1               | 2                         | 2.0                 | 4.2                |
+| Communications    | 0                 | 0               | 2                         | 1.0                 | 0.0                |
```

Comment:
We should remove lines from this table since they are `0`. If you want, we can make a couple of different tables for different workloads (e.g. lab - Observation heavy, telehealth- communications heavy, etc.)